src/common/options/mds.yaml.in

```diff
@@ -1563,6 +1563,18 @@ options:
   max: 4_K
   flags:
   - runtime
+- name: mds_use_global_snaprealm_seq_for_subvol
+  type: bool
+  level: advanced
+  default: false
```

Comment:
Should this be enabled by default to preserve existing behaviour?

Reply:
Makes sense. I will change this.

Reply:
Then we must ensure the ODF sets this to `false` ?

Reply:
yes, in one way or the other.